core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -294,6 +294,12 @@ public enum Property {
           + " user-implementations of pluggable Accumulo features, such as the balancer"
           + " or volume chooser.",
       "2.0.0"),
+  GENERAL_CACHE_MANAGER_IMPL("general.block.cache.manager.class",
+      "org.apache.accumulo.core.file.blockfile.cache.lru.LruBlockCacheManager", PropertyType.STRING,
+      "Specifies the class name of the block cache factory implementation."
+          + " Alternative implementation is"
+          + " org.apache.accumulo.core.file.blockfile.cache.tinylfu.TinyLfuBlockCacheManager.",
```

Comment:
Why change the default impl from LruBlockCacheManager to TinyLfuBlockCacheManager?

Reply:
The default is `LruBlockCacheManager`, I just copied the code from `TSERV_CACHE_MANAGER_IMPL`. The default was changed to the TinyLfuBlockCacheManager in a later release IIRC.